cli/cmd/dashboard.go

```diff
@@ -55,6 +74,26 @@ var dashboardCmd = &cobra.Command{
 	},
 }
 
+func isDashboardAvailable(kubeAPI k8s.KubernetesApi) (bool, error) {
+	client, err := kubeAPI.NewClient()
+	if err != nil {
+		return false, err
+	}
+	fullUrl, err := kubeAPI.UrlFor(controlPlaneNamespace, "/services/web")
```

Comment:
Could we do HTTP get on [`url`](https://github.com/runconduit/conduit/pull/289/files#diff-eeca73aa97366eceef4a2b10208faed2R38) instead of re-generating it?